packages/data/src/index.js

```diff
@@ -91,6 +91,25 @@ export { combineReducers };
  */
 export const select = defaultRegistry.select;
 
+/**
+ * Given the name of a registered store, returns an object that matches the store selectors.
```

Comment:
This can be reworded:

"Given the name of a registered store, returns an object containing the store's selectors pre-bound to state so that you only need to supply additional arguments, and modified so that they return promises that resolve to their eventual values, after any resolvers have ran."